doc/apiary/v2/fiware-ngsiv2-rc-2016_10_31.apib

```diff
@@ -0,0 +1,2027 @@
+FORMAT: 1A
+HOST: http://orion.lab.fiware.org
+TITLE: FIWARE-NGSI v2 Specification RC-2016_10
+DATE: 31 Oct 2016
+VERSION: stable
+PREVIOUS_VERSEION: RC-2016_05
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/2602/commits/e960821dbcb3e68125dabe80d153b42cf62860b1